doc/nrf/drivers/wifi/nrf700x/nrf700x.rst

```diff
@@ -76,9 +76,66 @@ Configuration
 
 The nRF70 Series Wi-Fi driver has the following configuration options:
 
+KConfig based
+^^^^^^^^^^^^^
+
 .. options-from-kconfig::
    :show-type:
 
+DTS based
+^^^^^^^^^
+
+Maximum transmit power achieved on a nRF70 device based product depends on frequency band and operating channel. This varies from chip to chip as well as over different PCB designs. Multiple calibrations and checks are implemented to get consistency across channels and devices. However, these values have dependency on PCB design which may result in EVM (Error Vector Magnitude) and Spectral mask failures. To avoid this problem, a provision is made in software for user to specify the power ceiling at which EVM and Spectral mask is met for a given PCB design and are build-time parameters made available to driver through DTS overlay file.
+
+Below is an example of the DTS overlay file
+
+.. code-block:: devicetree
+
+	/
+	{
+		nrf70_tx_power_ceiling: nrf70_tx_power_ceiling_node {
+			status = "okay";
+			compatible = "nordic,nrf700x-tx-power-ceiling";
+			max-pwr-2g-dsss = <0x54>;
+			max-pwr-2g-mcs0 = <0x40>;
+			max-pwr-2g-mcs7 = <0x40>;
+			max-pwr-5g-low-mcs0 = <0x38>;
+			max-pwr-5g-low-mcs7 = <0x38>;
+			max-pwr-5g-mid-mcs0 = <0x38>;
+			max-pwr-5g-mid-mcs7 = <0x38>;
+			max-pwr-5g-high-mcs0 = <0x38>;
+			max-pwr-5g-high-mcs7 = <0x38>;
+		};
+
+	};
+
+
+The following parameters (8-bit unsigned values) are defined in the DTS overlay board files
+
+.. list-table:: DTS file parameters
+   :header-rows: 1
+
+   * - DTS parameter
+     - Description
+   * - max-pwr-2g-dsss
+     - Transmit power ceiling for DSSS data rate in 0.25dBm steps. This is applicable for all DSSS data rates.
```

Comment:
Please write the second sentence in the new line for all such instances. Also, add space between the numerical value and unit symbol.
```suggestion
     - Transmit power ceiling for DSSS data rate in 0.25 dBm steps.
       This is applicable for all DSSS data rates.
```

Reply:
Made suggested changes.